registrar/registrar_test.go

```diff
@@ -256,6 +258,50 @@ var _ = Describe("Registrar.RegisterRoutes", func() {
 		})
 	})
 
+	Context("on startup", func() {
+		BeforeEach(func() {
+			port := 8080
+			rrConfig.Routes = []config.Route{
+				{
+					Name: "my route 1",
+					Port: &port,
+					URIs: []string{
+						"my uri 1.1",
+						"my uri 1.2",
+					},
+					Tags: map[string]string{
+						"tag1.1": "value1.1",
+						"tag1.2": "value1.2",
+					},
+					// RegistrationInterval is > the wait period in our Eventually() to ensure we've triggered on initial Run()
```

Comment:
Thanks for adding this comment. The long RegistrationInterval would be confusing otherwise. If you wanted to take a "no comments" approach, you could also do it like this:
```go
longIntervalToEnsureEventsAreTriggeredOnInitialRun := 10 * time.Second
rrConfig.Routes := {
...
  RegistrationInterval: longIntervalToEnsureEventsAreTriggeredOnInitialRun
```

In this case, I prefer the comment